index.html

```diff
@@ -1,197 +1,202 @@
+        <img class="header__pictures" src="img/rdl2.png" alt="burning man">
+        <img class="header__pictures" src="img/rdl3.png" alt="jumping man">
+        <img class="header__pictures" src="img/rdl1.png" alt="lara croft">
+        <img class="header__pictures" src="img/rdl2.png" alt="burning man">
+        <img class="header__pictures" src="img/rdl3.png" alt="jumping man">
+        <img class="header__pictures" src="img/rdl1.png" alt="lara croft">
+        <img class="header__pictures" src="img/rdl2.png" alt="burning man">
+        <img class="header__pictures" src="img/rdl3.png" alt="jumping man">
+        <img class="header__pictures" src="img/rdl1.png" alt="lara croft">
+        <img class="header__pictures" src="img/rdl3.png" alt="jumping man">
+        <img class="header__pictures" src="img/rdl1.png" alt="lara croft">
+        <div class="adminpanel">Тут будет <br> админпанель</div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="left-container">
+      <nav class="left-nav">
+        <h1>Фильмы онлайн</h1>
+        <ul class="left-nav__list">
+          <li class="left-nav__items"><a class="left-nav__href" href="">Трейлеры 2022</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Комедии</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Боевики</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Драмы</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Ужасы</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Фентези</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Мистика</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Военные</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Триллеры</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Криминал</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Семейные</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Детективы</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Фантастика</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Мелодрамы</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Мультфильмы</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Приключения</a></li>
+          <li class="left-nav__items what-more">а так же</li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Документальное кино</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Многосерийное кино</a></li>
+          <li class="left-nav__items"><a class="left-nav__href" href="">Музыкальные клипы</a></li>
+        </ul>
```

Comment:
`а так же` is a part of list? Seems to me no, so maybe it would be better to put it into `<p>` outside of list and separate a new `<ul>` with three another `<li>`

```suggestion
        <ul class="left-nav__list">
          <li class="left-nav__items"><a class="left-nav__href" href="">Трейлеры 2022</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Комедии</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Боевики</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Драмы</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Ужасы</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Фентези</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Мистика</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Военные</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Триллеры</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Криминал</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Семейные</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Детективы</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Фантастика</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Мелодрамы</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Мультфильмы</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Приключения</a></li>
        </ul>
        
        <p class="left-nav__items what-more">а так же</p>
          
        <ul class="left-nav__list">
          <li class="left-nav__items"><a class="left-nav__href" href="">Документальное кино</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Многосерийное кино</a></li>
          <li class="left-nav__items"><a class="left-nav__href" href="">Музыкальные клипы</a></li>
        </ul>
```